packages/amplify-provider-awscloudformation/src/utils/admin-helpers.ts

```diff
@@ -185,6 +185,10 @@ export const adminBackendMap: {
     amplifyAdminUrl: 'https://eu-north-1.admin.amplifyapp.com',
     appStateUrl: 'https://prod.eu-north-1.appstate.amplifyapp.com',
   },
+  'eu-south-1': {
+    amplifyAdminUrl: 'https://eu-south-1.admin.amplifyapp.com',
```

Comment:
Note: this is not available (yet). There's a bit of a circular dependency going on where the Amplify Studio depends on the amplifybackend service, which depends on Amplify CLI being available.

This means that Studio login will temporarily not work for the eu-south-1 region. The good news is, whether this login happens or not depends on the below appState url, which is also not available yet, so CLI users will not be prompted to log in until appState is enabled.